main/Helper.cpp

```diff
@@ -1128,12 +1136,18 @@ int SetThreadName(std::thread::native_handle_type thread, const char *name)
 	strncpy(name_trunc, name, sizeof(name_trunc));
 	name_trunc[sizeof(name_trunc)-1] = '\0';
 	return pthread_setname_np(thread, name_trunc);
```

Comment:
while there, there is a bug
`return pthread_setname_np(thread, name_trunc);` -> `return pthread_setname_np(thread, "%s", (void *)name_trunc);`

Reply:
Fixed.